### DISCUSSION TITLE
Support for sequelize

### DISCUSSION THREAD

**Discussion by Rich43** [TypedProject/ts-express-decorators#1945] [Q&A]
Hi, I use sequelize in my project. Are there any integrations in tsed for sequelize?

**Comment by Romakita** [comment]
Hello @Rich43,

Sorry for the late answer. Support Sequelize officially isn't planned and isn't really necessary.

By using [Async Provider](https://tsed.io/docs/custom-providers.html#use-async-factory) you can implement sequelize connection. You can use ts-sequelize to use decorators ;)

See you
Romain
